extension/src/openvic-extension/singletons/MenuSingleton.cpp

```diff
@@ -698,7 +698,10 @@ Dictionary MenuSingleton::get_topbar_info() const {
 	// TODO - colonial power info
 	ret[colonial_power_available_key] = 0;
 	ret[colonial_power_max_key] = 0;
-	ret[colonial_power_tooltip_key] = String {};
+	ret[colonial_power_tooltip_key] =
+		// TODO - remove these examples eventually
+		// make_rules_tooltip(country->get_rule_set()) + get_tooltip_separator() +
+		make_modifier_effects_tooltip(country->get_modifier_sum().get_value_sum());
```

Comment:
This is debugging/testing code for modifiers. It does not show colonial power.
So don't merge to master?

Reply:
I was thinking it would be nice for users to be able to see evidence of the modifier calculating system if they check out the game as it currently stands, but it's not that important